README.md

```diff
@@ -162,7 +162,8 @@ The documentation of all commands can be found in the comments of
 find them. All commands can be prefixed with the attribute `#[verbose]`
 to get an idea of what they are doing.
 
-See also the `#[log]` attribute in the "Plan B" section below.
+For debugging purposes, passing the attribute `#[log]` or `#[log(raw)]`
+to a HB command prints Coq commands which are equivalent to its effect.
```

Comment:
```suggestion
For debugging purposes, passing the attribute `#[log]` or `#[log(raw)]`
to a HB command prints Coq commands which are equivalent to its effect.
The elpi meta-data cannot be generated as Coq code though, which means the
displayed command cannot be a replacement for what HB generated.
```
@palmskog do you find this informative enough?

Reply:
Maybe also teaching purposes?